src/webcomponents/helperfunctions/jsonToCol.ts

```diff
@@ -0,0 +1,124 @@
+export const jsonToCol = (languages: string): any => {
+  let datas: any = JSON.parse(languages);
+  let dataArray: any = [];
+  let keys: any = [];
```

Comment:
do not prepare all the variables at the beginning of the function (use functional approach)

Reply:
Any suggation what to do?

Reply:
Let's inspire by:
```
const columns = mapLangToSheetColumns(languages) 
const data = mapLangToSheetData(languages)
```